File: python_scripts/savestate_test.py
import savestate
import emu

print("PY: Start savestate_test.py")

while(True):
    if emu.framecount() == 15 * 60:
        ss = savestate.object(1)
        print("PY: SAVE", ss)
        savestate.save(ss)
    
    # Repeats every 5 seconds, framecount is also loaded on load state 
    if emu.framecount() == 20 * 60:
        print("PY: LOAD", ss)
        savestate.load(ss)
    
    emu.frameadvance()

File: python_scripts/savestate_test.lua
print("Start savestate_test.lua")

ss = nil

while (1) do
    if (emu.framecount() == (15 * 60)) then
        ss = savestate.object(1)
        print("Save", ss)
        savestate.save(ss)
    end

    if (emu.framecount() == (20 * 60)) then
        print("Load", ss)
        savestate.load(ss)
    end

    emu.frameadvance()
end